app/assets/stylesheets/shared/newsletter-subscribe.scss

```diff
@@ -0,0 +1,57 @@
+.newsletter-form {
+  display: flex;
+  width: 100%;
+
+  input {
+    border-bottom-left-radius: $form-border-radius;
+    border-bottom-right-radius: 0;
+    border-top-left-radius: $form-border-radius;
+    border-top-right-radius: 0;
+
+    flex-grow: 1;
+    height: 40px;
+    vertical-align: top;
+  }
+
+  button {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: $base-border-radius;
+    border-top-left-radius: 0;
+    border-top-right-radius: $base-border-radius;
+
+    display: table-cell;
+    font-size: 1em;
+    height: 40px;
+    line-height: 1.3;
+    margin: 0;
+    padding: 0.25em 1em;
+    vertical-align: top;
+
+    i.fa {
+      color: white;
+      margin: 0;
+    }
+  }
+}
+
+.loading {
+  margin: 10% auto;
+  border-bottom: 4px solid #fff;
+  border-left: 4px solid #fff;
+  border-right: 4px solid $blue;
+  border-top: 4px solid $blue;
+  border-radius: 100%;
+  height: 25px;
+  width: 25px;
+  @include animation(spin .6s infinite linear);
+}
+
+@include keyframes(spin) {
+  0% {
+    @include transform(rotate(0deg));
+  }
```

Comment:
It seems like this is followed by an empty line?  Maybe just randomly add white space til hound stops crying.